tests/cypress/integration/reportRepair/tenantOrLeaseholder.spec.js

```diff
@@ -44,9 +45,12 @@ describe('tenant or leaseholder', () => {
       });
 
       context('When a user selects: No', () => {
```

Comment:
This test is testing multiple things and depends on `enableLeaseholderFlow()` - it needs to be rewritten as two different tests each testing for specific redirects.